packages/sui-lint/repository.config.js

```diff
@@ -0,0 +1,15 @@
+const RULES = {
+  OFF: 0,
+  WARNING: 1,
+  ERROR: 2
+}
+
+module.exports = {
+  plugins: ['sui'],
+  rules: {
+    'sui/node-version': RULES.WARNING,
```

Comment:
Having some doubts about keeping the prefix `sui/` in these rules... 🤔

Reply:
sure, we can use anyone that we prefer. Any suggestion ?!